doc/api/buffer.md

```diff
@@ -82,24 +80,24 @@ and replaced by separate `Buffer.from()`, [`Buffer.alloc()`], and
 *Developers should migrate all existing uses of the `new Buffer()` constructors
 to one of these new APIs.*
 
-* [`Buffer.from(array)`] returns a new `Buffer` containing a *copy* of the provided
-  octets.
-* [`Buffer.from(arrayBuffer[, byteOffset [, length]])`][`Buffer.from(arrayBuffer)`]
-  returns a new `Buffer` that *shares* the same allocated memory as the given
-  [`ArrayBuffer`].
-* [`Buffer.from(buffer)`] returns a new `Buffer` containing a *copy* of the
+* [`Buffer.from(array)`] returns a new `Buffer` that *contains a copy* of the
+  provided octets.
+* [`Buffer.from(arrayBuffer[, byteOffset [, length]])`]
```

Comment:
This line break makes the link not to be rendered.

Reply:
[Still the issue)](https://github.com/Trott/io.js/blob/e20d30eb0304d9d0f3f2cf44ecb0145caa8d8173/doc/api/buffer.md#bufferfrom-bufferalloc-and-bufferallocunsafe)